### PR TITLE
Shayan/Enable Chart Indicators | Comparisons on Mobile Responsive

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

File: sass/components/_studylegend.scss [hunk 1]
```diff
@@ -241,6 +249,11 @@
             background: themed('IndicatorInfoBg');
         }
 
+        &__content {
+            background: #ff0000;
```
Comment (balakrishna-binary) [inline sass/components/_studylegend.scss:253]: do we have to consider dark theme?

File: sass/components/_dialog.scss [hunk 2]
```diff
@@ -259,5 +262,39 @@
...
+        }
     }
-}
+    .cq-modal--settings {
+        .sc-dialog {
+            width: 320px;
+            left: calc(50% - 160px);
+        }
+    }
+}
```
Comment (balakrishna-binary) [inline sass/components/_dialog.scss:300]: new line?

Comment (shayan-binary-2) [reply sass/components/_dialog.scss]: @balakrishna-binary just modified

File: sass/components/_studylegend.scss (hunk 1 above)
Comment (shayan-binary-2) [reply sass/components/_studylegend.scss]: @balakrishna-binary oops my bad, just fixed it.

**Review by zaki-binary — APPROVED** [review]

**Pull request merged**